sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/OciImageManifest.java

```diff
@@ -35,28 +33,28 @@ public final class OciImageManifest implements JsonSerializable<OciImageManifest
     /*
      * Schema version
      */
-    private int schemaVersion = 2;
+    private int schemaVersion;
```

Comment:
Don't we want the default to be `2`?

Reply:
we did,  but got feedback from .NET architects that we should either make it a constant or let users configure it.  Since service allows to pass it and does not provide a default value, we reverted to default value transformation.